### PR TITLE
Replace forecastio library with the original forecast API

### PR DESCRIPTION
**Pull request by admiral96**

### CODE REVIEW

File: app/weather_parser.py [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
 from config import FORECAST_API_KEY
 
 
 class WeatherForecast:
 
-    def generate_forecast(self, latitude, longitude, frequency):
-        forecast = self.get_forecast_request(latitude, longitude)
-        #  ---------We won't support hourly and daily forecasts yet---------
-        return forecast.currently()
+    def connect(self, lat, lng, time=None):
```
Comment (dashmug) [inline app/weather_parser.py]: Please give this a better name. Its purpose is not just to _connect_ but to retrieve API results, right?

Comment (admiral96) [inline app/weather_parser.py]: Yep, got it.

**Comment by admiral96** [comment]
Code updated @dashmug

**Comment by dashmug** [comment]
@admiral96 In your future PRs, squash them only _after_ review is done or when it's ready to merge. It makes it easier for reviewers to see what you changed between commits.

**Comment by admiral96** [comment]
Got it @dashmug

**Comment by admiral96** [comment]
Code updated @dashmug

**Pull request merged**